src/macros/src/lib.rs

```diff
@@ -14,6 +15,15 @@ pub fn cpu_abi(_: TokenStream, item: TokenStream) -> TokenStream {
         .into()
 }
 
+#[proc_macro_attribute]
```

Comment:
Do we really need an attribute macro? I think just a derive macro should be enough.